### PR TITLE
[AirFlow][refactor] change periodic expression and add long running dag

### PR DESCRIPTION
**Pull request by wuchaochen**
## What is the purpose of the change

1. Make task periodic expression same to cron table expression.
2. Dag with periodic task or subscribed event task is long running dag.

### CODE REVIEW

File: flink-ai-flow/lib/airflow/tests/contrib/jobs/test_event_based_scheduler.py [hunk 1]
```diff
@@ -266,34 +267,35 @@ def run_trigger_dag_function(self):
                 if len(tes) > 0:
                     break
                 else:
-                    client.trigger_parse_dag()
+                    client.trigger_parse_dag('../../dags/test_run_trigger_dag.py')
```
Comment (SteNicholas) [inline flink-ai-flow/lib/airflow/tests/contrib/jobs/test_event_based_scheduler.py:270]: Does this need to move the dags directory to jobs directory?

File: flink-ai-flow/lib/airflow/tests/contrib/jobs/test_event_based_scheduler.py [hunk 2]
```diff
@@ -398,22 +400,41 @@ def test_stop_dag(self):
...
+            with create_session() as session:
+                tes = session.query(TaskExecution).filter(TaskExecution.dag_id == 'single',
+                                                          TaskExecution.task_id == 'task_1').all()
+                if len(tes) > 2:
+                    break
+                else:
+                    time.sleep(1)
+        self.client.send_event(StopSchedulerEvent(job_id=0).to_event())
+
+    def test_run_cron_periodic_task(self):
```
Comment (SteNicholas) [inline flink-ai-flow/lib/airflow/tests/contrib/jobs/test_event_based_scheduler.py:414]: Add the test cases for the incorrect cron expression config.

File: flink-ai-flow/lib/airflow/tests/contrib/jobs/test_event_based_scheduler.py [hunk 3]
```diff
@@ -398,22 +400,41 @@ def test_stop_dag(self):
...
-                if len(tes) > 1:
+                if len(tes) > 2:
                     break
                 else:
                     time.sleep(1)
         self.client.send_event(StopSchedulerEvent(job_id=0).to_event())
 
-    def test_run_periodic_task(self):
-        t = threading.Thread(target=self.run_periodic_task_function, args=())
+    def test_run_interval_periodic_task(self):
```
Comment (SteNicholas) [inline flink-ai-flow/lib/airflow/tests/contrib/jobs/test_event_based_scheduler.py:433]: Add the test cases for the incorrect interval expression config.

**Review by SteNicholas — CHANGES_REQUESTED** [review]

File: flink-ai-flow/lib/airflow/tests/contrib/jobs/test_event_based_scheduler.py (hunk 1 above)
Comment (wuchaochen) [reply flink-ai-flow/lib/airflow/tests/contrib/jobs/test_event_based_scheduler.py:270]: All test dag files are in the dag directory.

**Review by SteNicholas — APPROVED** [review]

File: flink-ai-flow/lib/airflow/airflow/contrib/jobs/periodic_manager.py [hunk 4]
```diff
@@ -42,48 +48,60 @@ def shutdown(self):
...
-                return 
+            interval_expr: str = periodic_config['interval']
+            interval_items = interval_expr.split(',')
+            if len(interval_items) != 5:
+                raise ValueError('The interval expression {} is incorrect format, follow the pattern: '
+                                 'weeks,days,hours,minutes,seconds.'.format(interval_items))
+            temp_list = []
+            is_zero = True
+            for item in interval_items:
+                if item is None or '' == item.strip():
```
Comment (jiangxin369) [inline flink-ai-flow/lib/airflow/airflow/contrib/jobs/periodic_manager.py:86]: I suppose None or '' are illegal interval and will raise exception by strip(). We don't need to parse them as 0.

**Review by SteNicholas — APPROVED** [review]

**Pull request merged**